source/_components/sensor.fints.markdown

```diff
@@ -0,0 +1,78 @@
+---
+layout: page
+title: "FinTS Sensor"
+description: "Instructions on how to use the FinTS sensor."
+date: 2018-03-25 13:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Sensor
+ha_release: 0.67
+ha_iot_class: "Local Push"
+logo: home-assistant.png
+---
+
+With the FinTS sensor, you can fetch your account information from your bank. This only works with banks that support the FinTS (aka. HBCI) standard. The FinTS standard used by many German banks. So if you do not have a German bank account, this will most likely not work for you. To find out if your bank supports FinTS, check the bank's website or call their hotline.
+
+For each account you have with the bank, a separate sensor is created. If you have several accounts with a bank, you can select which ones you want to have, and you can also give the accounts a name.
+
+{% raw %}
+```yaml
+# Example configuration.yaml entry
+sensor:
+ - platform: fints
+    name: <give the bank a name> 
```

Comment:
`name` seems to be an optional configuration variable.

Please remove all optional parameters from initial example. We'd like to keep the first example as minimal as possible.

Reply:
done

Reply:
Hm, yes sorry. But now its in...